ui/prometheus-plugin/src/plugins/prometheus-time-series-query/PrometheusTimeSeriesQueryEditor.tsx

```diff
@@ -57,11 +57,12 @@ export function PrometheusTimeSeriesQueryEditor(props: PrometheusTimeSeriesQuery
       />
       <TextField
         fullWidth
-        label="Series Name Format"
+        label="Name"
+        placeholder="Tip: Use {{variable_name}}. Example: {{instance}} will be replaced with values such as 'webserver-123' and 'webserver-456'."
+        helperText="Set the name for each series in the legend and the tooltip."
         value={format ?? ''}
         onChange={(e) => handleFormatChange(e.target.value)}
         onBlur={handleFormatBlur}
-        margin="dense"
```

Comment:
Screenshot: 

<img width="871" alt="Screenshot 2023-04-26 at 1 55 24 PM" src="https://user-images.githubusercontent.com/2584129/234700256-c5fb8d1f-f8eb-4b63-b3d8-1ffb4b895657.png">

@sjcobb Am I right to say `variable_name`? Or is `label_name`? (i.e. should they use the name of the label on the metric, or should they use the name of one of the variables that they created)?

Reply:
label_name is correct! There is overlap since a lot of times the variables can match but what’s inside the handlebars {{ }} syntax is just regex that has nothing to do with variable context. (We do most likely also need to also start supporting template variable `$` syntax too, there is a bug ticket about this)

Reply:
Maybe "Legend Name" for the TextField label? I think that's the only variation we haven't discussed yet? 😂

Reply:
Pushed up 'Name' -> 'Legend Name' change, we can always adjust again later and I might propose in Matrix chat that we change the JSON from `series_name_format` to `legend_format` since it's best to do the breaking change sooner rather than later while we don't have many users

Reply:
Also made the 'variable_name' to 'label_name' change in my commit [here](https://github.com/perses/perses/pull/1104/commits/74dcafe0e14e50f08ce8914e990a8627b70e04b1)

Reply:
I like it - I also just pushed up to this branch because I had not reloaded this page yet to see your changes, but should be fine since I think it's already deleted

Reply:
I like Legend Name a lot actually